packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -154,6 +154,14 @@ const PersonalDetails = ({
             />
         );
     };
+
+    /*
+    In most modern browsers, setting autocomplete to "off" will not prevent a password manager from asking the user if they would like to save username and password information, or from automatically filling in those values in a site's login form.
+    check this link https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields
+    */
+    // for dropdowns use 'new-password'
+    const autocomplete_value = 'new-password';
```

Comment:
doesn't  'none' making more sense since we are using it for different fields with different names?

Reply:
@mahdiyeh-deriv I understand what you mean, but I did like [here](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password) for preventing autofilling with autocomplete. But your case works for me too, so I can change it to 'none'.